### PR TITLE
Release/1.2.0

### PR DESCRIPTION
**Pull request by darrelmiller**
Releasing Milestone [1.2.0](https://github.com/microsoft/OpenAPI.NET/milestone/3)

### CODE REVIEW

File: src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+using Microsoft.OpenApi.Readers.Interface;
+using SharpYaml;
+using SharpYaml.Serialization;
+
+namespace Microsoft.OpenApi.Readers
+{
+    /// <summary>
+    /// Service class for converting contents of TextReader into OpenApiDocument instances
+    /// </summary>
+    public class OpenApiTextReaderReader : IOpenApiReader<TextReader, OpenApiDiagnostic>
```
[the remark this answers is not in the document]
Comment (darrelmiller) [reply src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs:17]: Yeah, it is debatable.  It is a OpenAPIReader that takes a TextReader as the input.  Therefore it is an OpenApi[TextReader]Reader.

File: src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        /// Reads the stream input and parses it into an Open API document.
+        /// </summary>
+        /// <param name="input">TextReader containing OpenAPI description to parse.</param>
+        /// <param name="diagnostic">Returns diagnostic object containing errors detected during parsing</param>
+        /// <returns>Instance of newly created OpenApiDocument</returns>
+        public OpenApiDocument Read(TextReader input, out OpenApiDiagnostic diagnostic)
+        {
+            YamlDocument yamlDocument;
+
+            // Parse the YAML/JSON
```
[the remark this answers is not in the document]
Comment (darrelmiller) [reply src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs:40]: The TextReader input could contain Yaml or JSON.  That is what we are parsing.  We are loading the Yaml or JSON text into a YamlDocument object.

[the remark this answers is not in the document]
Comment (VitaliyKurokhtin) [reply src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs:40]: JSON is technically YAML after all...

[the remark this answers is not in the document]
Comment (darrelmiller) [reply src/Microsoft.OpenApi.Readers/OpenApiTextReaderReader.cs]: I updated the comment for additional clarity

**Pull request merged**